src/app/components/page/page.component.ts

```diff
@@ -7,7 +8,23 @@ import { Component, OnInit } from '@angular/core';
 })
 export class PageComponent implements OnInit {
 
-  constructor() { }
+  @Input()
+  course: ICourse = {
+    id: 1,
+    title: '',
+    creationTime: 1,
+    duration: 1,
+    description: ''
+  };
+
+  @Output() destruction = new EventEmitter<any>();
+
+  destruct(): void {
+    console.log(this.destruction);
```

Comment:
![image](https://user-images.githubusercontent.com/11182310/136012156-1702f4d1-f374-405a-af20-0443bd12831a.png)
you should log id, why you log EventEmitter?